client/js/widgets/seat.js

```diff
@@ -92,7 +92,12 @@ class Seat extends Widget {
     }
   }
 
-  updateLinkedWidgets() {
-    widgetFilter(w=>w.get('onlyVisibleForSeat') || w.get('linkedToSeat') || w.get('type') == 'seat').forEach(wc=>wc.updateOwner());
+  updateLinkedWidgets(playerChanged) {
+    const scoreboard = widgetFilter(w => w.get('type') == 'scoreboard');
+    for(const board of scoreboard)
+      board.updateTable();
```

Comment:
Feel free to revert, but in https://github.com/ArnoldSmith86/virtualtabletop/pull/1381/commits/1ae186ebc23fffe0df0a6185405e7aaff325f961 by having this call updateTable() instead of applyDeltaToDOM(null) we don't need to specially detect null in scoreboard.applyDeltaToDOM as it's only called with updates.

Reply:
Looks good to me.